app/templates/partials/flash_messages/archive_service_confirmation_message.html

```diff
@@ -0,0 +1,20 @@
+{% if platform_admin %}
+      <h3 class="govuk-body govuk-!-font-weight-bold">Before you delete ‘{{service_name}}’, confirm the user: </h3>
+           <ul class="govuk-list govuk-list--bullet">
+             <li>has given the exact name of the service they want to delete</li>
+             <li>understands they can no longer receive text messages (if they use this feature)</li>
+           </ul>
+
+      <h3 class="govuk-body govuk-!-font-weight-bold">You also need to:</h3>
+           <ul class="govuk-list govuk-list--bullet">
+              <li>check the user has the ‘Manage settings, team and usage’ permission for this service</li>
+              <li>ask a team member with the ‘Manage settings, team and usage’ permission to confirm the request</li>
+              <li>check if the service has exceeded its annual allowance, in case we need to raise an invoice</li>
+           </ul>
```

Comment:
I was thinking about other ways to structure these instructions.

There’s a lot the user needs to think about/do, and probably an order of priority in which to do it.

### Version 1


>Before you delete this service:
>
>1. Confirm the exact name of the service the user wants to delete.
>1. Check the user has the ‘Manage settings, team and usage’ permission for this service.
>
> If the service can receive text messages, tell the user that their phone number will stop working.
>
> Do not delete the service until you have:
>
>* confirmation from another team member with the ‘Manage settings, team and usage’ permission
>* checked if we need to raise an invoice

### Version 2
This one condenses all the most important stuff into a numbered list, rather than 2 lists of competing priority.

>Before you delete this service:
>
>1. Confirm the exact name of the service the user wants to delete.
>1. Check the user has the ‘Manage settings, team and usage’ permission for this service.
>1. Confirm the request with another team member who has the ‘Manage settings, team and usage’ permission.
>1. Check if we need to raise an invoice.
>
>If the service can receive text messages, tell the user that their phone number will stop working.

Reply:
After discussing with Saima on Slack, we’re suggesting this for our MVP.

We need to monitor this over time and check in with the team to understand if we want to:

* add more content
* remove anything
* reintroduce the nuance of Saima’s original 2 list approach
* revisit the idea of taking users to a separate page

```suggestion
      <h3 class="govuk-body govuk-!-font-weight-bold">Before you delete ‘{{service_name}}’: </h3>
           <ol class="govuk-list govuk-list--number">
             <li>Confirm the exact name of the service the user wants to delete.</li>
             <li>Check the user has the ‘Manage settings, team and usage’ permission for this service.</li>
             <li>Confirm the request with another team member who has the ‘Manage settings, team and usage’ permission.</li> 
             <li>Check if we need to raise an invoice.</li>           
           </ol>
           
       <p class="govuk-body">If the service can receive text messages, tell the user that their phone number will stop working.</p>
```

Reply:
Happy with the succinctness of the content and, once in use,  would like feedback from the team on how well the guidance is/isn't working.